### DISCUSSION TITLE
Need help configuring Renovate

### DISCUSSION THREAD

**Discussion by vedxp** [renovatebot/renovate#18824] [Request Help] [answered]
Here's the link of `renovate.json` in my repo: [Visit File](https://github.com/VedxP/aw-some/blob/main/renovate.json).

I just want Renovate to auto merge the PR upon passing deployment on Vercel.

I configured the file myself, but Renovate opened [this issue](https://github.com/VedxP/aw-some/issues/3).

Need help in fixing the configuration I did wrong.

**Comment by viceice** [comment, marked as answer]
read the docs section for package rules. you need at least one matchxxx it will excludexxx property.
